examples/features/stats_monitoring/client/main.go

```diff
@@ -0,0 +1,97 @@
+// Binary client is an example client.
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/stats"
+
+	pb "google.golang.org/grpc/examples/features/proto/echo"
+)
+
+var addr = flag.String("addr", "localhost:50051", "the address to connect to")
+
+// *statsHandler implements [stats.Handler](https://pkg.go.dev/google.golang.org/grpc/stats#Handler) interface
+type statsHandler struct{}
+
+// TagRPC can attach some information to the given context.
+// The context used for the rest lifetime of the RPC will be derived from the returned context.
+func (st *statsHandler) TagRPC(ctx context.Context, stat *stats.RPCTagInfo) context.Context {
+	log.Printf("[TagRPC] RPC request send to: %s", stat.FullMethodName)
+	return ctx
+}
+
+// Note: All stat fields are read-only
+func (st *statsHandler) HandleRPC(ctx context.Context, stat stats.RPCStats) {
+	switch stat := stat.(type) {
+	case *stats.Begin:
+		log.Printf("[HandleRPC] [%T] Request sending process started at: %s", stat, stat.BeginTime.Format(time.StampMicro))
```

Comment:
Please print the complete `Begin` struct, and the same comment applies for other cases as well. Thanks.